File: Respostas/Bee_1073/1073.c
#include <stdio.h>
#include <math.h>

int main() {
	int N;
	
	scanf("%d", &N);
	
	for (int i = 1; N >= i; i++) {
		if ((i % 2) == 0)
			printf("%d^2 = %.0f\n", i, pow(i, 2));
	}
	
	return 0;
}

File: Respostas/Bee_1073/1073.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		int N;
		
		Scanner input = new Scanner(System.in);
		
		N = input.nextInt();
		
		for (int i = 1; N >= i; i++) {
			if ((i % 2) == 0)
				System.out.printf("%d^2 = %.0f\n", i, Math.pow(i, 2));
		}
	}
}